libs/designsystem/src/lib/components/data-table/index.ts

```diff
@@ -0,0 +1,7 @@
+export { TableComponent } from './table/table.component';
+export { TableTheadComponent } from './thead/thead.component';
+export { TableTbodyComponent } from './tbody/tbody.component';
+export { TableTfootComponent } from './tfoot/tfoot.component';
+export { TableTrComponent } from './tr/tr.component';
```

Comment:
It would be great if we can remove the T and call them:
 
TableComponent
TableHeadComponent
TableBodyComponent
TableFootComponent
TableRowComponent